go.sum

```diff
@@ -370,6 +370,8 @@ github.com/bmatcuk/doublestar/v4 v4.6.0/go.mod h1:xBQ8jztBU6kakFMg+8WGxn0c6z1fTS
 github.com/bradleyfalzon/ghinstallation/v2 v2.1.0 h1:5+NghM1Zred9Z078QEZtm28G/kfDfZN/92gkDlLwGVA=
 github.com/bradleyfalzon/ghinstallation/v2 v2.1.0/go.mod h1:Xg3xPRN5Mcq6GDqeUVhFbjEWMb4JHCyWEeeBGEYQoTU=
 github.com/bwesterb/go-ristretto v1.2.3/go.mod h1:fUIoIZaG73pV5biE2Blr2xEzDoMj7NFEuV9ekS419A0=
+github.com/cam72cam/hcl/v2 v2.0.0-20240416130056-03228b26f391 h1:X8ZCyI4kvTg3sm0colNG6x+rz4Ntb76ZoFwUyiuE3k0=
```

Comment:
I will eat some humble pie here: I've now read the PR description where you discuss this

> As this may take some time for the HCL Team to review, I will likely move the forked code under the OpenTofu umbrella before this is merged.

Reply:
Yeah, let's please move it to the opentofu org.